source/androidx.activity/activity/Additions/ActivityResultContracts.cs

```diff
@@ -0,0 +1,182 @@
+using System;
+using Android.Runtime;
+using Java.Interop;
+using Java.Lang;
+
+namespace AndroidX.Activity.Result.Contract 
+{
+	internal delegate IntPtr _JniMarshal_PPLL_L(IntPtr jnienv, IntPtr klass, IntPtr p0, IntPtr p1);
+
+	// Metadata.xml XPath class reference: path="/api/package[@name='androidx.activity.result.contract']/class[@name='ActivityResultContracts']"
+	//[global::Android.Runtime.Register ("androidx/activity/result/contract/ActivityResultContracts", DoNotGenerateAcw=true)]
+	public sealed partial class ActivityResultContracts // : global::Java.Lang.Object 
```

Comment:
It might be nice to leave a comment of why this was needed so it might be removed later if it is no longer needed.

Reply:
Will do!